docs/en/seatunnel-engine/logging.md

```diff
@@ -92,6 +92,24 @@ SeaTunnel provides an API for querying logs.
 
 For more details, please refer to the [REST-API](rest-api-v2.md).
 
+## SeaTunnel Log Configuration
+
+### Scheduled deletion of old logs
+
+SeaTunnel supports scheduled deletion of old log files to prevent disk space exhaustion. You can add the following configuration in the `seatunnel.yml` file:
+
+```yaml
+seatunnel:
+  engine:
+    history-job-expire-minutes: 1440
+    telemetry:
+      logs:
+         scheduled-deletion-enable: false
```

Comment:
Should this configuration be deleted? Keeping it as part of the job history information may cause disk accumulation.

cc @Hisoka-X